### PR TITLE
Homepage changes

### PR DESCRIPTION
**Pull request by jaidevanoop**
Tell me what else needs to be changed.

screenshot from 2016-02-21 13 44 46

### CODE REVIEW

**Comment by abhijithanilkumar** [comment]
Do we need the side bar?

**Comment by jaidevanoop** [comment]
How do we access things like Message from the president? Also. Working on adding date and time to event before merging

**Comment by abhijithanilkumar** [comment]
The current one doesnt look good. Can we have something like sidebar in mobile apps? When we click on a button it shows up.

**Comment by jaidevanoop** [comment]
Yeah possible. This is just temporary. I didnt know what to do with the links exactly. The dates for events are up.

screenshot from 2016-02-21 14 39 50

**Comment by rohitvarkey** [comment]
@jaidevanoop Maybe use [this](http://materializecss.com/side-nav.html) for the sidenav? Let's keep it collapsed by default.

**Comment by jaidevanoop** [comment]
@rohitvarkey Cool. But the jquery initialization isn't working.

**Comment by rohitvarkey** [comment]
Are you putting the scripts in the 'script' block?

**Comment by jaidevanoop** [comment]
@rohitvarkey Yeah. The dropdown initialization works. sideNav doesn't. Using dropdown for normal view. sideNav for mobile screens

**Comment by jaidevanoop** [comment]
@rohitvarkey @abhijithanilkumar Got it fixed. now what do i do for the homepage?

File: src/static/site/css/main.css [hunk 1]
```diff
@@ -133,7 +133,7 @@ body>.navbar-transparent {
 .text-page {
     padding: 120px 0;
 }
-
+s
```
Comment (rohitvarkey) [inline src/static/site/css/main.css]: remove this

File: src/templates/base.html [hunk 2]
```diff
@@ -24,28 +24,46 @@
 </head>
 <body>
   {% block navbar %}
-  <div class="navbar-fixed">
```
Comment (rohitvarkey) [inline src/templates/base.html]: Don't remove this. Let's keep the navbar fixed on top

**Comment by rohitvarkey** [comment]
Also what do you think is nicer? SideNav or dropdown?

**Comment by jaidevanoop** [comment]
@rohitvarkey Navbar remains fixed only. Where is the difference? Also, i was thinking, dropdown should do for large screens and sidenav for mobiles. What say?

**Comment by rohitvarkey** [comment]
I like the way the dropdown links have been organized into sections. Can we do the same for the sidebar, leaving alone the misc classification? That is, can we have "Student Council" as a one item, which on clicking shows us the links under it and similarly for "What's New" ?

I think the sidenav is cooler! Anyway, we can decide that tomorrow in the meeting.

And when I scroll the navbar goes out of focus. I'd like it to be fixed on top always.

**Comment by jaidevanoop** [comment]
@rohitvarkey Check now. I think i got it done. We'll discuss the dropdown/sidenav tomorrow?

**Comment by rohitvarkey** [comment]
Yeah! I have a few suggestions that we'll go over tomorrow. Good stuff!

**Comment by rohitvarkey** [comment]
Just had a chat with Ansu regarding the homepage. Couple of things she pointed out were:

- [ ] Picture thumbnails for news and blog
- [ ] Better fonts

**Comment by jaidevanoop** [comment]
@rohitvarkey Links added to navbar. A few pages haven't been created. Also, there arent links for teh footer. Changed static as well. Check it out

**Comment by rohitvarkey** [comment]
@jaidevanoop The second background picture is too bad. We need to find some solution for that. Maybe revert to the plain background?

**Pull request merged**